packages/cli/src/util/loggerConsoleTransport.ts

```diff
@@ -0,0 +1,56 @@
+import winston, {Logger} from "winston";
+// eslint-disable-next-line import/no-extraneous-dependencies
+import {LEVEL} from "triple-beam";
+import {LogLevel} from "@lodestar/utils";
+
+interface DefaultMeta {
+  module: string;
+}
+
+interface LogInfo extends DefaultMeta {
+  [LEVEL]: LogLevel;
+}
+
+export class ConsoleDynamicLevel extends winston.transports.Console {
+  private readonly levelByModule = new Map<string, LogLevel>();
+  private readonly defaultLevel: LogLevel;
+
+  // Define property from TransportStream that's not in the types
+  private readonly levels!: Record<LogLevel, number>;
+  private parent?: Logger;
+
+  constructor(opts: {defaultLevel: LogLevel} & winston.transports.ConsoleTransportOptions) {
+    super(opts);
+
+    this.defaultLevel = opts.defaultLevel;
+
+    // Set level and parent to undefined so that underlying transport logs everything
+    this.level = undefined;
+  }
+
+  setModuleLevel(module: string, level: LogLevel): void {
+    this.levelByModule.set(module, level);
+  }
+
+  deleteModuleLevel(module: string): boolean {
+    return this.levelByModule.delete(module);
+  }
+
+  _write(info: LogInfo, enc: BufferEncoding, callback: (error?: Error | null | undefined) => void): void {
```

Comment:
A bit intrusive into Winston mechanics but this is the easiest solution to have custom module levels by far from the few options I've tried